peekaboo/toolbox/file.py

```diff
@@ -66,9 +66,6 @@ class Filetools(object):
     """ Parent class, defines interface to various file tools. """
     def get_report(self, sample):
         """ Return filetools report or create if not already cached. """
-        if sample.filetools_report != None:
```

Comment:
Maybe squash this into the very first commit and never introduce it to then remove it again?

Reply:
Addressed in #143.